apiary.apib

```diff
@@ -1040,6 +1080,108 @@ API endpoints that Operation Code's Rails backend makes available to its React f
         {
             errors: "Some error message"
         }
+## User | Update [/api/v1/users{?education_level,mentor,scholarship_info,employment_status,company_name,company_role,volunteer,branch_of_service,interests}]
+
++ Parameters
+
+  + education_level (string, optional) - Users highest level of education in the form of a string
+  + mentor (boolean, optional) - Is User requesting mentor in the form of a boolean
+  + scholarship_info (boolean, optional) - Is User interested in scholarship info in the form of a boolean
+  + employment_status (string, optional) - Users employment status in the form of a string
+  + company_name (string, optional) - Users name of employer in the form on a string
+  + company_role (string, optional) - Users role at company in the form of a string
+  + volunteer (boolean, optional) - Is User wanting to volunteer in the form of a boolean
+  + branch_of_service (string, optional) - Users branch of service in the form of a string
+  + interests (string, optional) - User interests in the form of a string.
+
+### Update user record [PATCH]
+
++ Request (application/json)
+
+    + Headers
+
+            Authorization: Bearer Access-Token
+
+    + Body
+
+            {
+                education_level: "College Graduate" ,
+                mentor: "true",
+                scholarship_info: "true",
+                employment_status: "Employed",
+                company_name: "Comcast",
+                company_role: "Developer",
+                volunteer: "false",
+                branch_of_service: "Air Force",
+                interests: "Python, Python, Python, Python"
+            }
+
++ Response 200 (application/json)
+
+    + Body
+
+            {
+                first_name: "Mike",
+                last_name: "Johnson",
+                email: "dreams@operationcode.org",
+                zip: "80020",
+                password: "hunter2",
+                mentor: "true",
+                education_level: "College Graduate",
+                scholarship_info: "true",
+                employment_status: "Employed",
+                company_name: "Comcast",
+                company_role: "Developer",
+                volunteer: "false",
+                branch_of_service: "Air Force",
+                interests: "Python, Python, Python, Python"
+            }
+
++ Response 422 (application/json)
+
+    + Body
+
+            {
+                errors: "Some error message"
+            }
+
+## User | Verify [/api/v1/users/profile/verify{?verified}]
+
++ Parameters
+
+  + verified (boolean, required) - User has verified military affiliation status through ID.me
```

Comment:
The `param` here is only [access_token](https://github.com/OperationCode/operationcode_backend/blob/master/app/controllers/api/v1/users_controller.rb#L33)